### PR TITLE
JAMES-1673 Remove last warnings on Eclipse

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: server/data/data-hbase/pom.xml [hunk 1]
```diff
@@ -115,7 +115,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>13.0</version>
```
Comment (mbaechler) [inline server/data/data-hbase/pom.xml]: nop, it won't build

File: mailbox/zoo-seq-provider/pom.xml [hunk 2]
```diff
@@ -66,7 +66,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>13.0</version>
```
Comment (mbaechler) [inline mailbox/zoo-seq-provider/pom.xml]: nop, it won't build

File: mailbox/hbase/pom.xml [hunk 3]
```diff
@@ -35,7 +35,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>11.0.2</version>
```
Comment (mbaechler) [inline mailbox/hbase/pom.xml]: nop, it won't build

File: server/data/data-hbase/pom.xml (hunk 1 above)
Comment (aduprat) [inline server/data/data-hbase/pom.xml]: I've check the compilation (without test) on both Java-6 & Java-8

Comment (mbaechler) [inline server/data/data-hbase/pom.xml]: ok, to be more specific, tests won't work

Comment (aduprat) [inline server/data/data-hbase/pom.xml]: OK

File: mailbox/zoo-seq-provider/pom.xml (hunk 2 above)
Comment (aduprat) [inline mailbox/zoo-seq-provider/pom.xml]: OK

File: mailbox/hbase/pom.xml (hunk 3 above)
Comment (aduprat) [inline mailbox/hbase/pom.xml]: OK

**Comment by rouazana** [comment]
:+1:

**Comment by mbaechler** [comment]
:+1:

**Pull request closed without merging**